src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java

```
@@ -0,0 +1,437 @@
+        return Arrays.asList(new Coverage(), new RMSMappingQuality(), new ReadPosRankSumTest(), new MappingQualityRankSumTest());
+
+    }
+
+    public void onTraversalStart() {
+        VCFHeader inputHeader = getHeaderForVariants();
+        final Set<VCFHeaderLine> inputHeaders = inputHeader.getMetaDataInSortedOrder();
+
+        final Set<VCFHeaderLine> headerLines = new HashSet<>(inputHeaders);
+        // Remove GCVFBlocks, legacy headers, and annotations that aren't informative for single samples
+        headerLines.removeIf(vcfHeaderLine -> vcfHeaderLine.getKey().startsWith(GVCFWriter.GVCF_BLOCK) ||
+                (vcfHeaderLine.getKey().equals("INFO")) && infoFieldAnnotationKeyNamesToRemove.contains(((VCFInfoHeaderLine)vcfHeaderLine).getID()));
+
+        headerLines.addAll(getDefaultToolVCFHeaderLines());
+
+        genotypingEngine = createGenotypingEngine(new IndexedSampleList(inputHeader.getGenotypeSamples()));
+        createAnnotationEngine();
+
+        headerLines.addAll(annotationEngine.getVCFAnnotationDescriptions(false));
+        headerLines.add(VCFStandardHeaderLines.getInfoLine(VCFConstants.DEPTH_KEY));   // needed for gVCFs without DP tags
+        headerLines.add(GATKVCFHeaderLines.getInfoLine(GATKVCFConstants.RAW_QUAL_APPROX_KEY));
+        headerLines.add(GATKVCFHeaderLines.getInfoLine(GATKVCFConstants.VARIANT_DEPTH_KEY));
+        headerLines.add(GATKVCFHeaderLines.getInfoLine(GATKVCFConstants.MAPPING_QUALITY_DEPTH));
+
+        if ( dbsnp.dbsnp != null  ) {
+            VCFStandardHeaderLines.addStandardInfoLines(headerLines, true, VCFConstants.DBSNP_KEY);
+        }
+
+        VariantContextWriter writer = createVCFWriter(outputFile);
+
+        try {
+            vcfWriter = new GVCFWriter(writer, GVCFGQBands, PLOIDY_TWO);
+        } catch ( IllegalArgumentException e ) {
+            throw new CommandLineException.BadArgumentValue("GQBands", "are malformed: " + e.getMessage());
```

Could you also propagate the underlying exception as the cause here?  It's always best to do.